### DISCUSSION TITLE
Metadata panel - set default order for all tables

### DISCUSSION THREAD

**Discussion by xorock** [dbeaver/dbeaver#8639] [Q&A] [answered]
#### System information: 
-Win10 x64
- DBeaver version 7.0.4

#### Connection specification:
- PostgreSQL

#### Describe your question:
Is it possible to set a default sort order by specific header in the metadata panel? For example, I almost always use sort by "Name" column so every time I open new table I have to do extra click.

**Comment by kseniiaguzeeva** [comment, marked as answer]
No, you have to do the sorting when you open Metadata panel.
Please reopen the ticket or create a new one, if you have any other questions.
